### ISSUE TITLE
Rubick doesn't spell steal techies minefield sign

### ISSUE THREAD

**Issue by Fourcups** [jeffhill/Dota2#277]
### Ability name

Spell steal

### Description

Title

### Example Match ID (and possibly Timestamp)

n/a

**Comment by NMOTH** [comment]
It is not a bug. Because it is no longer a spell. Just a cosmetic.

**Comment by nickwong1** [comment]
Spell Steal has never been able to steal Minefield Sign even before the rework iirc. Likely to be intended!

**Comment by TheZett** [comment]
> Spell Steal has never been able to steal Minefield Sign even before the rework iirc. Likely to be intended!

Incorrect, you used to get Minefield Sign when stealing the old proximity/land mines and Stasis Traps.

Why else would rubick have his own unique Rubick sign?

https://www.reddit.com/r/DotA2/comments/2faq8x/with_the_new_minefield_sign_ability_rubick_gets/

**Comment by nickwong1** [comment]
> Incorrect, you used to get Minefield Sign when stealing the old proximity/land mines and Stasis Traps.

 TIL, wow I do not remember seeing this in-game, and thank you for correcting me @TheZett ! 👍 🙏

**Comment by NMOTH** [comment]
Please close this.

**Comment by AmanovAkbar** [comment]
> It is not a bug. Because it is no longer a spell. Just a cosmetic.

It is a press able skill in his skillbar tho.

**Comment by jere535** [comment]
> > It is not a bug. Because it is no longer a spell. Just a cosmetic.
> 
> It is a pressable skill in his skillbar tho.

Some abilities just aren't stealable, mostly abilities that would otherwise be free spell steal counters for no good reason.

**Comment by TheZett** [comment]
> Please close this.

This is a valid complaint.
While the Minefield Sign should not be stealable on its own, it should be granted as well when stealing Techies’ Proximity Mines (like it used to be the case a few years ago).

**Comment by Daar543** [comment]
> > Please close this.
> 
> This is a valid complaint. While the Minefield Sign should not be stealable on its own, it should be granted as well when stealing Techies’ Proximity Mines (like it used to be the case a few years ago).

That would nerf his aghanim scepter though
